workshops/Tkinter Workshop/README.md

```diff
@@ -0,0 +1,249 @@
+---
+name: "To Quote or Not To Quote?"
+description: "Let's build a slideshow of quotes with Tkinter and Python!"
+author: "@bezlin6mechminerz"
+---
+
+Having a user interface is very useful because it gives both a good interactive experience for the user and makes it much more visually appealing. So today we will see how to make a simple user interface with python in this workshop.
+
+The workshop will look something like this.
+
+![ezgif.com-video-to-gif-3.gif](https://cloud-rbupe5aas.vercel.app/0ezgif.com-video-to-gif-2.gif)
+
+So we will be making a quotes hub which displays quotes.
+
+View a [live demo](https://repl.it/@bezlin/trialtkinterpy-2#main.py)
+
+View the [final code](https://repl.it/@bezlin/trialtkinterpy-2#main.py)
+
+This workshop will take about 20 minutes.
+
+## Getting started
+
+We will be using [Tkinter](https://docs.python.org/3/library/tk.html), a python library to develop our user interface. This project requires [Repl.it.](https://repl.it) We will be using it for coding. It is awesome because you can code online. Just follow this link and start coding!
+
+To get started, [click here](https://repl.it/languages/Tkinter). Your coding environment will be ready in a few seconds!
+<img width="1440" alt="Screenshot 2020-09-18 at 5 39 06 PM" src="https://cloud-o8q46nkko.vercel.app/screenshot_2020-09-25_at_9.23.06_pm.png">
+
+#### What is Tkinter?
+
+Tkinter is a library in python used to make cool User Interfaces(UI). What the user sees on the screen to interact with the program is called a user interface. Here we will use it to make our Quotes Hub.
+
+## Code and explanation.
+
+Lots of beginners make endless changes to their code and expect it to miraculously work right away. The problem with this approach is that it stacks one problem on top of another, and it becomes difficult to figure what went wrong. So Code along this will allow you to learn practical skills while achieving something!
```

Comment:
“ So Code along this will allow you to learn practical skills” grammar/doesn’t make sense

Reply:
changed!